script.js

```diff
@@ -0,0 +1,135 @@
+
+const searchButton = document.querySelector('#search-button');
+let searchBar = document.querySelector('#search-input');
+
+function generateCards(movieObject){
+    let star = document.createElement('span');
+    star.classList.add('star');
+    let starContent = document.createTextNode('⭐️ ');
+    star.appendChild(starContent);
+    // document.body.appendChild(star);
+
+
```

Comment:
recommend using a helper function to make code more DRY (to create/append elements)